doc/help/settings.asciidoc

```diff
@@ -223,6 +223,7 @@
 |<<scrolling.bar,scrolling.bar>>|Show a scrollbar.
 |<<scrolling.smooth,scrolling.smooth>>|Enable smooth scrolling for web pages.
 |<<session.default_name,session.default_name>>|Name of the session to save by default.
+|<<session.lazy_restore,session.lazy_restore>>|Load a restored tab as soon as it takes focus.
```

Comment:
This is an unrelated change, and it's already like this in `master` - did you base your PR on some older `master` branch or something?

Reply:
Right, I didn't update the code since my last pull request.